_topic_map.yml

```diff
@@ -1299,12 +1299,12 @@ Topics:
     File: migrating-applications-with-cam-4-1-4
   - Name: Troubleshooting
     File: troubleshooting-4-1-4
-- Name: Migrating Openshift Container Platform 4.2 to 4.3
+- Name: Migrating from Openshift Container Platform 4.2
   Dir: migrating_4_2_4
   Distros: openshift-enterprise
   Topics:
-  - Name: Migrating application workloads from OpenShift Container Platform 4.2 to 4.3
-    File: migrating-application-workloads-4-2-4
+  - Name: Migration tools and prerequisites
+    File: migration-tools-prerequisites-4-2-4
```

Comment:
Leaving this one here but applies to all three files that are changing. 

Is this rename completely changing the meaning of what the original file was doing? I don't know the full details but is migrating application workloads section now covering tools and prereqs for migrations? If yes, does it means we no longer use the application workloads?

Sorry, just trying to figure out what the intent is here.

Reply:
No, the new name finally matches the contents of the file. :-)  When this assembly was first created, it was a bit of a mess, so "application workloads" was a meaningless catch-all title. However, it's also a misleading and inaccurate title, because all of migration is about migrating application workloads. This file morphed into a tools, concepts, and prerequisites section, so the new title makes a lot more sense.

Reply:
@apinnick Ok, the title change is ok, but does the file name needs to change?

Reply:
I suppose the file name doesn't need to change. I could revert the file name. Is there any reason not to change it?

Reply:
Every assembly filename change or a folder move requires a new redirect in already published docs :) I am trying to keep these to the minimum to not overload our redirects server. If you feel that by not changing the name of the file causes issues in the overall structure of these sections, I am ok with that.

Reply:
Ah, OK. I'm reverting the file names.